README.md

```diff
@@ -18,6 +18,7 @@ Install dependencies.
 
     $ ./bin/pip install Cython==0.29.9
     $ ./bin/pip install kivy==1.11.1
+    $ ./bin/pip install pyyaml
```

Comment:
I guess at some point we want to migrate to more conventional dependency install method.
Let's keep it in mind for later

Reply:
sure, install convenience and instructions is planned with making dependencies optional